src/containers/App/Content.tsx

```diff
@@ -213,8 +215,11 @@ function GetCapabilities({children}: {children: React.ReactNode}) {
     useCapabilitiesQuery();
     const capabilitiesLoaded = useCapabilitiesLoaded();
 
+    useMetaCapabilitiesQuery();
+    const metaCapabilitiesLoaded = useMetaCapabilitiesLoaded();
```

Comment:
Did not dig deep but looks like if there is no meta metaCapabilitiesLoaded will always be falsy and loading will always be true?